src/fidesops/ops/models/manual_webhook.py

```diff
@@ -23,3 +27,45 @@ class AccessManualWebhook(Base):
     )
 
     fields = Column(MutableList.as_mutable(JSONB), nullable=False)
+
+    @property
+    def fields_schema(self) -> BaseSchema:
+        """Build a dynamic Pydantic schema from fields defined on this webhook"""
+
+        class Config:
+            extra = "forbid"
+
+        field_definitions: Dict[str, Any] = {
+            field["dsr_package_label"]: (Optional[str], None)
+            for field in self.fields or []
+        }
+
+        ManualWebhookValidationModel = create_model(  # type: ignore
+            __model_name="ManualWebhookValidationModel",
+            __config__=Config,
+            **field_definitions,
+        )
+        return ManualWebhookValidationModel
+
+    @property
+    def empty_fields_dict(self) -> Dict[str, None]:
+        """Return a dictionary that maps defined dsr_package_labels to None
+
+        Returned as a default if no data has been uploaded for a privacy request.
+        """
+        return {
+            key: None
+            for key in (self.fields_schema.schema().get("properties") or {}).keys()
+        }
+
+    @classmethod
+    def get_enabled(cls, db: Session) -> List["AccessManualWebhook"]:
```

Comment:
I haven't used this type notation before- what's the difference between wrapping the type in quotes vs just using `List[AccessManualWebhook]` ?

Reply:
This is only because this class method is within the AccessManualWebhook class itself. AccessManualWebhook isn't bound yet when we're evaluating the function return .  Wrapping it in quotes lets us delay when the type gets evaluated.  I think in later versions of Python this delayed evaluation will become the default.

Reply:
Oh wow, didn't know about this, thanks for the explanation!